.github/workflows/build.yml

```diff
@@ -39,6 +39,6 @@ jobs:
         with:
           path: .tox
           # bump version prefix to fully reset caches
-          key: v1-tox-${{ matrix.python-version }}-${{ hashFiles('tox.ini') }}
+          key: v2-tox-${{ matrix.python-version }}-${{ hashFiles('tox.ini') }}
```

Comment:
The tox envs are incorrectly cached otherwise because the `hashFiles()` doesn't account for modified `setup.py` files.

Reply:
What breaks if you change to `hashFiles('tox.ini', '**/setup.py')` ?